Libraries/LibGUI/GTextEditor.cpp

```diff
@@ -498,6 +501,7 @@ void GTextEditor::redo()
     }
 
     m_undo_stack_index--;
+    on_change();
```

Comment:
It's not safe to call `on_change` directly, since the user of this `GTextEditor` may not have installed a hook.
Instead, call `did_change()`.